manifest.json

```diff
@@ -60,18 +84,30 @@
   },
   "contributions": [
     {
-      "id": "service-endpoint",
+      "id": "google-cloud-endpoint-ui",
+      "displayname": "Google Cloud Platform service endpoint UI",
+      "type": "ms.vss-endpoint.service-endpoint-ui",
+      "targets": [
+        "ms.vss-endpoint.endpoint-ui-catalog"
+      ],
+      "properties": {
+        "file": "main.html"
+      }
+    },
```

Comment:
As both contribution google-cloud-endpoint-ui and google-cloud-endpoint-type depends on each other (Google contribution has to have UI contribution with it as we are deleting rendering code form VSTS code), I am not getting the significance of using different manifest file. Please brief in case I missed something,

Reply:
See my earlier comment about why I want a separate folder and manifest.json. As for the link between the endpoint-type and the endpoint-ui, I think it is fine to separate them. The endpoint-type is connected to the endpoint-ui, but it is also connected to build tasks. Keeping that central point of connection at the root level makes sense.

Reply:
I will try it in next iteration.

Reply:
We have moved all ui files to new folder gcp-ui-build-task except this file. Reason is: the way we are interacting with the contribution is requiring main.html to be at root path. We can't change it as changing in our code will require to change other contribution's main.html to move to different location. Hope you can allow this single file to be presented in root manifest